code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -96,7 +96,6 @@
 
 /obj/structure/closet/secure_closet/security/PopulateContents()
 	..()
-	new /obj/item/clothing/suit/armor/vest(src)
 	new /obj/item/clothing/suit/armor/vest/alt/sec(src)
 	new /obj/item/clothing/head/helmet/sec(src)
 	new /obj/item/radio/headset/headset_sec(src)
```

Comment:
should we remove the non-bowman headset as well?

Reply:
Not sure if we'd want that it also would be out the scope of the PR Since every head locker has normal+bowman headsets maybe a maintainers opinion could help here